src/basic/generate-af-list.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-cpp -dM -include sys/socket.h - </dev/null | \
+$1 -E -dM -include sys/socket.h - </dev/null | \
```

Comment:
It was exactly for things like `generate-cap-list.sh` that I said you should pass the cmd_array in the end, and use `shift` to fetch the N headers that you want to pass as arguments. Something like this:

```sh
H1="$1"
H2="$2"
shift 2
"$@" [...] -include "$H1" -include "$H2"
```

Doesn't really matter I guess. :)